### DISCUSSION TITLE
This is gonna take some time...

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#59] [Week 2 - Instructional Pros and Cons]
I'll say what I liked first, this was that exercises were given to us, I feel that all too often teachers will have the students learn the material on their own and there are no actual exercises built into the learning flow. That being said...
I found all of the exercises to be extremely hard to navigate and figure out, I am very new to coding languages and scripts, so I found myself doing a lot of extra tutorials and watching a lot of extra videos. 
In the end, I did like doing all of the exercises, I just wish that had a little more guidance associated with them.

**Comment by gabebott** [comment]
I agree. As someone that is very new to this language as well, I get your pain. I think that these sights could use a little more guidance and helplines for those that are true beginners.

**Comment by kassyoveson** [comment]
I struggled with the coding homework for the first few weeks. It was very hard, and I also used a lot of YouTube videos to get through the tutorials. It was not always explained well. Even if I had an example of what they were looking for would have been helpful.
